NAMESPACE

```diff
@@ -24,27 +24,44 @@ export(module_aglu_L205.ag_prodchange_cost_input_DISABLED)
 export(module_aglu_L2051.ag_prodchange_cost_input_irr_DISABLED)
 export(module_aglu_L2071.ag_water_irr_DISABLED)
 export(module_aglu_L221.land_input_1_DISABLED)
+<<<<<<< HEAD
```

Comment:
@bpbond I don't understand how the conflicts are created.  For example a `UCD_trn_data_CORE.csv` file has an error in it because now there is a line with `<<<<<<HEAD` and  `==========` and `<<<<MASTER`.
I didn't touch these files or change anything.  Is it happening when I merge?